unlock-app/src/components/interface/locks/CheckoutUrl/index.tsx

```diff
@@ -423,9 +458,13 @@ export const CheckoutUrlPage = () => {
                   description:
                     'Select the locks that you would like to featured in this configured checkout modal',
                   disabled: !hasSelectedConfig,
+                  loading,
                   children: (
                     <LocksForm
-                      onChange={onAddLocks}
+                      onChange={(fields: any, isValid = true) => {
+                        if (!isValid) return // trigger update form only when is valid to prevent update auto-update with invalid values
```

Comment:
Please add a logging statement when the config is invalid as it should not happen.

Reply:
as before, prevent auto-save when form is invalid, eg. `url invalid`

<img width="600" alt="Screenshot 2023-07-25 at 10 46 58" src="https://github.com/unlock-protocol/unlock/assets/20865711/dcb8d869-dcb7-4054-8500-148f91c70abc">